Dashboard/Views/ConnectionWindow.axaml

```diff
@@ -0,0 +1,57 @@
+<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        x:Class="Dashboard.ConnectionWindow"
+        Title="ConnectionWindow"
+        Width="800"
+        Height="500">
+    <DockPanel>
+
+        <!-- Main Content Grid -->
+        <Grid DockPanel.Dock="Top">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="2*"/>
+                <ColumnDefinition Width="3*"/>
+            </Grid.ColumnDefinitions>
+
+            <StackPanel Grid.Column="0" Margin="20">
+                <!-- Connection Type -->
+                <TextBlock Text="Connection Type" Margin="0, 10, 0, 5"/>
+                <ComboBox Width="200">
+                    <ComboBoxItem>Serial Port</ComboBoxItem>
+                    <ComboBoxItem>TCP</ComboBoxItem>
+                    <ComboBoxItem>File (.CSV)</ComboBoxItem>
+                    
+                </ComboBox>
+
+                <!-- Serial Port -->
+
+                 <TextBlock Text="Serial Port" Margin="0, 10, 0, 5"/>
+                <StackPanel Orientation="Horizontal">
+
+                    <ComboBox x:Name="cmbSerialPort" Width="200" Margin="0, 10, 0, 5"></ComboBox>
```

Comment:
Can you populate this box from the start, so we don't need to click the refresh at start